README.md

```diff
@@ -29,32 +46,32 @@ Specify a name of the file `File::Find` is ought to look for. If you pass a stri
 type
 ----
 
-Given a type, `find()` will only return files being the given type. The available types are `file`, `dir` or `symlink`.
+Given a type, `find()` will only return files being the given type. The available types are `file`, `dir`, or `symlink`.
 
 exclude
 -------
 
-Exclude is meant to be used for skipping certain big and uninteresting directories, like '.git'. Neither them nor any of their contents will be returned, saving a significant amount of time.
+Specify a regex (or any other smartmatchable type) to exclude files / directories from the search.
 
-The value of `exclude` will be smartmatched against each IO object found by File::Find. It's recommended that it's passed as an IO object (or a Junction of those) so we avoid silly things like slashes vs backslashes on different platforms.
+recursive
+---------
+
+By default, `find` will recursively traverse a directory tree, descending into any subdirectories it finds. This behavior can be changed by setting `recursive` to a false value. In this case, only the first-level entries will be processed.
 
 keep-going
 ----------
 
 Parameter `keep-going` tells `find()` to not stop finding files on errors such as 'Access is denied', but rather ignore the errors and keep going.
 
-recursive
----------
-
-By default, `find` will recursively traverse a directory tree, descending into any subdirectories it finds. This behaviour can be changed by setting `recursive` to a false value. In this case, only the first level entries will be processed.
+**Note: This parameter is currently broken and does not affect the search in any way.**
 
 Perl's File::Find
 =================
 
-Please note, that this module is not trying to be the verbatim port of Perl's File::Find module. Its interface is closer to Perl's File::Find::Rule, and its features are planned to be similar one day.
+Please note that this module is not trying to be the verbatim port of Perl's File::Find module. Its interface is closer to Perl's File::Find::Rule, and its features are planned to be similar one day.
 
 CAVEATS
 =======
 
-List assignment is eager in Raku, so if You assign `find()` result to an array, the elements will be copied and the laziness will be spoiled. For a proper lazy list, use either binding (`:=`) or assign a result to a scalar value (see SYNOPSIS).
+List assignment is eager in Raku, so if you assign `find()` result to an array, the elements will be copied and the laziness will be spoiled. For a proper lazy list, assign a result to a scalar value (see SYNOPSIS).
```

Comment:
Any objections against binding?

Reply:
Not I.

Reply:
I meant "use either binding (`:=`) or" disappeared from the code. (The capital y was probably a "polonism")

Reply:
Corrected to original wording.